wordpress-mobile/WordPress-Android
# [Dependency Cache] Enable Dependency Cache on CI [without GRADLE_RO_DEP_CACHE]

Related: #21540

## Description:

This is a test only PR that attempts to test another dependency cache mechanism, one that does NOT dependent on the read-only cache ([GRADLE_RO_DEP_CACHE](https://docs.gradle.org/current/userguide/dependency_caching.html#sec:shared-readonly-cache)), and per project.

-----

## To Test:



`N/A`

> [!TIP]
> To verify that `GRADLE_RO_DEP_CACHE` is **NOT** enabled make sure you don't see this `Shared read-only dependency cache is an incubating feature.` log when Gradle launches for a CI build ([example](https://buildkite.com/automattic/wordpress-android/builds/20727#0193df28-3e2a-46e6-98e9-166835d11945/226-243)).

-----

## Regression Notes: `N/A`

-----

## PR Submission Checklist:


-----

## Testing Checklist (strike-out the not-applying and unnecessary ones): `N/A`
